### DISCUSSION TITLE
Does anybody know how to fix rectangle working on visual studio code?

### DISCUSSION THREAD

**Discussion by thisway8788** [rxhanson/Rectangle#850] [Q&A]
I just updated visual studio code and control + option key does not work for me
I used to resize and move the vs tab with control + option key but it does not work on vs code
Does anybody have this kind of issue?

**Comment by rxhanson** [comment]
This comes up from time to time (ex: https://github.com/rxhanson/Rectangle/issues/182)

Unfortunately, there's nothing that can be done on Rectangle's side for apps that fail to respond to the macOS accessibility window move/resize API. Restarting VSCode usually fixes the issue, but I suggest filing an issue with VSCode for it.
